repo_resource/test_check.py

```diff
@@ -32,6 +32,18 @@ def setUp(self):
                 'name': 'aosp_device_fixed.xml'
             }
         }
+        self.demo_manifests_to_rewrite_source = {
+            'source': {
+                'url':
+                'https://unreachable-github.com/makohoek/demo-manifests.git',
+                'revision': 'main',
+                'name': 'aosp_device_fixed.xml',
+                'insteadOf': {
+                    'https://github.com':
```

Comment:
Can we reverse the way the dictionary is structured?

Right now we have: REPLACEMENT:ORIGINAL.

What happens in the following scenario:
```
'insteadOf': {
    'https://github.com': 'https://unreachable-github.com',
    'https://gitlab.com': 'https://unreachable-github.com'
}
```
?

So key=replacement and value=original.

To me, it should be:
```
'insteadOf': {
    'https://unreachable-github.com': 'https://github.com'
}
```

Even if git uses the opposite. If we take above example:
```
git config --global url."https://github.com".insteadOf "https://unreachable-github.com"
```
https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf

Which translates into "use REPLACEMENT instead of ORIGINAL"

I still think that the dictionary should have key=ORIGINAL and value=REPLACEMENT

Reply:
@makohoek It sounds odd to me as well but I wanted to keep it identical to git so we keep the logic.
REPLACEMENT insteadOf ORIGINAL
If you really want to switch, I'd suggest to change the variable to rewrite / rewriteTo / rewriteUrl or something similar that matches the switch logic